File: aisile-sellergoods-interface/src/main/java/com/aisile/sellergoods/service/SellerService.java
package com.aisile.sellergoods.service;

import java.util.List;

import com.aisile.entity.PageResult;
import com.aisile.pojo.TbBrand;
import com.aisile.pojo.TbSeller;
import com.aisile.pojogroup.Specification;

public interface SellerService {

	//商家注册
	boolean add(TbSeller tbSeller);
	
	boolean updateStatus(String sellerId,String status);

	//查看商家（包括未注册的商家）
	PageResult findPage(int page, int rows, TbSeller tbSeller);
	
	//商家详情
	TbSeller findOne(String sellerId);
	
	
}

File: aisile-manager-web/src/main/webapp/js/service/sellerService.js

app.service("sellerService",function($http){
	
	this.search = function(page,rows,searchEntity){
		return $http.post('../seller/search.do?page='+page+'&rows='+rows,searchEntity)
	}
	
	this.findOne = function(sellerId){
		return $http.post('../seller/findOne.do?sellerId='+sellerId)
	}
	
	this.update = function(sellerId,status){
		return $http.post('../seller/update.do?sellerId='+sellerId+'&status='+status)
	}
	
})